decorator.rb

```diff
@@ -0,0 +1,12 @@
+require './nameable'
+
+class Decorator < Nameable
+  def initialize(nameable)
+    @nameable = nameable
```

Comment:
- [ ] Please make sure to make the `@nameable` instance variable accessible using the attr_accessor, in order for your code to work correctly.

Reply:
Thank you.